editions/tw5.com/tiddlers/macros/tag-pill_Macro.tid

```diff
@@ -14,7 +14,7 @@ The <<.def tag-pill>> [[macro|Macros]] generates a static tag pill showing a spe
 ;element-tag
 : The element name to be used for the pill (defaults to "span")
 ;element-attributes
-: Additional attributes for the pill element
+: Additional attributes for the pill element 
```

Comment:
Why is there a space needed?

Reply:
Oops! I had thought about rewriting the sentence, and missed that.